requirements.txt

```diff
@@ -267,6 +267,8 @@ googleapis-common-protos==1.60.0
     # via
     #   google-api-core
     #   grpcio-status
+greenlet==3.1.1
+    # via sqlalchemy
```

Comment:
I think this is changing because someone forgot to call `uv compile` after changing the requirements recently